columnflow/tasks/plotting.py

```diff
@@ -191,6 +178,27 @@ class PlotShiftedVariables(
     # default plot function
     plot_function_name = "columnflow.plotting.example.plot_shifted_variable"
 
+    per_process = luigi.BoolParameter(
+        default=False,
+        significant=True,
+        description="when True, one plot per process is produced; default: False",
+    )
+
+    legtitle = luigi.Parameter(
+        default=law.NO_STR,
+        significant=False,
+        description="sets the title of the legend; when empty and only one process is present in "
+        "the plot, the process_inst label is used; empty default",
+    )
```

Comment:
For consistency we should maybe name this `legend_title`? It could help to make this a little more verbose on the command line.

Reply:
Sure. I think this parameter could also be part of `PlotBase`, right? I'd just need to remove the plotting function dependent part of the description.

Reply:
Yeah, I think this makes sense in PlotBase.